packages/@react-aria/actiongroup/src/useActionGroup.ts

```diff
@@ -70,14 +71,17 @@ export function useActionGroup<T>(props: AriaActionGroupProps<T>, state: ListSta
     }
   };
 
-  let role = BUTTON_GROUP_ROLES[state.selectionManager.selectionMode];
+  let role: string | undefined = BUTTON_GROUP_ROLES[state.selectionManager.selectionMode];
+  if (isInsideAToolbar && role === 'toolbar') {
+    role = 'group';
+  }
```

Comment:
What is the advantage of using `role="toolbar"` in action group in the first place? Should we just always make it a "group"? I would really like to avoid this prop/context.

Reply:
I was mostly trying to keep it as it was already implemented, it does add to the announcement. It matters more for the radiogroup, but the rest could probably be group. I'll change it for now and we can debate it more when both examples are easily available for testing.

Reply:
mm one thing I could see being an issue, because it would no longer announce that it's a toolbar, is that users may not know they need to use arrow keys to navigate the group